packages/main/src/Dialog.ts

```diff
@@ -85,6 +85,10 @@ const ICON_PER_STATE: Record<ValueStateWithIcon, string> = {
  * <li>content - Used to style the content of the component</li>
  * <li>footer - Used to style the footer of the component</li>
  * </ul>
+ * <b>Note:</b> When a <code>ui5-bar</code> is used in the header or in the footer, you should remove the default dialog's paddings.
+ * <br>
+ * For more information see the sample "Bar in Header/Footer".
```

Comment:
Ask Ilhan if it's fine to refer a concrete sample from the JSDoc

Reply:
He is OK with this :)